backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/TestResultTest.java

```diff
@@ -30,6 +31,7 @@
   @Autowired private TestDataFactory _dataFactory;
   @Autowired private OrganizationService _orgService;
   @MockBean private SmsService _smsService;
+  @MockBean private CurrentTenantDataAccessContextHolder _tenantDataAccessContextHolder;
```

Comment:
**nitpick:** If it needs to be propagated into all of the GraphQL tests, should this Mock be installed in `BaseGraphqlTest` instead?

Reply:
That would be nice, but I couldn't find a way to override a MockBean added in one of the base classes.  Some of the tests in the api.graphql package do not seem to work with a request-scoped bean and I think it would be best not to use the mock in `ApiUserManagementTest`, so that's why it was left this way.  If you know of a way to fix this, I'd be happy to improve it.

Reply:
Ahh that makes sense; thanks!